### ISSUE TITLE
Windows 11 Edge browser doesn't display the chessboard GUI correctly in the editor. It's black.

### ISSUE THREAD

**Issue by Curious2021** [gbtami/pychess-variants#786]
**Exact URL where the bug happened**

**Steps to reproduce the bug**
As far as I can tell this bug happens on the Windows 11 desktop environment and only when using Microsoft Edge browser. When using Edge and you try editing a position using the GUI...the screen is black. You can see the pieces but not the board.
**What did you expect to happen?**
That I could use the website features normally.
**What happened instead?**
Can play using the  Edge browser on Windows 11 but cannot edit any position because Edge doesn't correctly render the GUI.
**Operating system and browser version**
Windows 11 latest build (beta channel)

**Comment by gbtami** [comment]
I have no Win11  to test it, but I suggest to clear browser cache. Edge is Chromium based, so it should be OK.

**Comment by Curious2021** [comment]
I have Firefox latest build on my Windows 11 system and it works fine and I never had to concern myself over the cache. This is a rendering issue due to CSS coding for the Edge browser.

**Comment by gbtami** [comment]
No. We have no Edge specific CSS. And Edge is based on Chromium rendering engine.
Have you tried to clear browser cache?

**Comment by Curious2021** [comment]
I just cleared the cache in Microsoft Edge in Windows 11 and it's still showing this issue with the GUI in the editor. Not the playing tab. So this tells me there's an issue with the CSS coding that's specific to Microsoft Edge.

**Comment by gbtami** [comment]
Works for me on Win10 and Edge Version 97.0.1072.62 (Official build) (64-bit)

**Comment by gbtami** [comment]
![kép](https://user-images.githubusercontent.com/479964/149431552-72053987-4f3a-4246-bd49-637142c8a021.png)

**Comment by gbtami** [comment]
Maybe your Edge from beta channel is also beta, idk.

**Comment by Curious2021** [comment]
That's probably it
